### PR TITLE
refactor: 💡 update accounts model with ldap account attributes

### PR DESCRIPTION
**Pull request by lisbet-alvarez**
✅ Closes: ICU-10050

:tickets: [Jira ticket](https://hashicorp.atlassian.net/browse/ICU-10050)

### CODE REVIEW

**Review by dvonanderson — APPROVED** [review]
LGTM

File: ui/desktop/electron-app/yarn.lock [hunk 1]
```diff
@@ -3269,7 +3269,7 @@ tar-stream@^1.5.2:
     to-buffer "^1.1.1"
     xtend "^4.0.0"
 
-tar@^6.0.5, tar@^6.1.11, tar@^6.1.2, tar@^6.1.9:
```
Comment (calcaide) [inline ui/desktop/electron-app/yarn.lock:3272]: @lisbet-alvarez interesting we are seeing a dependency change here when your changes does not add anything related 🤔

**Review by calcaide — APPROVED** [review]
Good work!! I left a none blocking comment

File: ui/desktop/electron-app/yarn.lock (hunk 1 above)
Comment (DhariniJeeva) [reply ui/desktop/electron-app/yarn.lock:3272]: I think the main is out of sync, I also got this update locally

**Review by DhariniJeeva — APPROVED** [review]

File: ui/desktop/electron-app/yarn.lock (hunk 1 above)
Comment (lisbet-alvarez) [reply ui/desktop/electron-app/yarn.lock:3272]: Yup, change happened when I used `yarn` in order to use `yarn test`

**Pull request merged**